app/components/volume/volumeController.js

```diff
@@ -16,13 +16,32 @@ function ($scope, $state, $transition$, VolumeService, Notifications) {
       $('#loadingViewSpinner').hide();
     });
   };
+  
+  function getVolumeDataFromContainer(container) {
+    return container.Mounts.find(function(volume) {
+      return volume.Name === $scope.volume.Id;
```

Comment:
I'd rather see the `volumeID` to be integrated as a second parameter of `getVolumeDataFromContainer()` instead of relying on the scope value here.